### PR TITLE
soopy since everyones been asking

### PR DESCRIPTION
**Pull request by Mqisty**

### CODE REVIEW

File: files/mods.json [hunk 1]
```diff
@@ -1398,6 +1398,31 @@
...
     },
+    {
+        "id": "soopy",
+        "nicknames": [
+            "soopyv2"
+        ],
+        "display": "SoopyV2",
+        "creator": "Soopyboo32",
+        "discordcode": "dfSMq96RSN",
+        "description": "A ChatTriggers module providing many unique and useful features for Hypixel SkyBlock.",
```
Comment (KTibow) [inline files/mods.json:1410]: but its not chattriggers is it

Comment (Mqisty) [reply files/mods.json:1410]: > but its not chattriggers is it

it is still a chattriggers module, the only thing the forge mod does is install the module lol

Comment (KTibow) [reply files/mods.json:1410]: the mod description is "Rendering optimisations for the chattriggers module soopyv2"
i dont think it downloads chattriggers into your mods folder right

Comment (Mqisty) [reply files/mods.json:1410]: > i dont think it downloads chattriggers into your mods folder right

it does not we'd probably have to find a way to make sure the user has chattriggers already installed/checked 

or we could just not do this at all its up to you and the other admins

Comment (MicrocontrollersDev) [reply files/mods.json:1410]: chattriggers has a warning box when installing. could do the same thing but for soopy saying that chattriggers needs to be installed too

**Comment by Wyvest** [comment]
apparently soopy is EOL sooo...

**Pull request closed without merging**